testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2879,6 +2879,7 @@ public void testInsert()
             assertQueryFails("INSERT INTO nation(nationkey) VALUES (42)", "This connector does not support inserts");
             return;
         }
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
```

Comment:
should be `supports_create_table_with_data` since even for `CTAS WITH NO DATA` query the code path being used is the CTAS code path.